code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -91,8 +91,8 @@
 //M4A3 45 //Inspired by the 1911
 
 /obj/item/weapon/gun/pistol/m1911
-	name = "\improper M4A3 service pistol (.45)"
-	desc = "A standard M4A3 chambered in .45. Has a smaller magazine capacity, but packs a better punch."
+	name = "\improper Concord FourFive service pistol"
+	desc = "(M4A3 .45) A C-45 service pistol. Chambered in .45. Hits harder than its 9mm sibling, the C-22 service pistol, in exchange for lower ammo capacity. Made by Armsdealer's Concord."
```

Comment:
Including the old name between parenthesis at the start of the description will make the transition just more confusing.

Reply:
Will be making quick adjustments.
Parenthesis with the old weapon name will be removed.
More numbers.
Less Concord.